src/spells/CESFork_GoerliRwaSpell.t.sol

```diff
@@ -725,7 +719,7 @@ contract DssSpellTest is DSTest, DSMath {
         assertEq(chainlog.getAddress("MIP21_LIQUIDATION_ORACLE"), addr.addr("MIP21_LIQUIDATION_ORACLE"));
     }
 
-    function testSpellIsCast_RWA007SGFWOFH1_INTEGRATION_BUMP() public {
+    function testSpellIsCast_CESFork_RWA007SGFWOFH1_INTEGRATION_BUMP() public {
```

Comment:
Shouldn't we stick to the leaner RWA007 naming convention instead of RWA007SGFWOFH1?

Reply:
you are right, i will do it after I redeploy wOFH contract, as current deployment use the old naming

Reply:
Also need to adjust addresses after that